requirements/requirements-convert-legacy-llama.txt

```diff
@@ -3,3 +3,4 @@ sentencepiece~=0.2.0
 transformers>=4.40.1,<5.0.0
 gguf>=0.1.0
 protobuf>=4.21.0,<5.0.0
+python-frontmatter~=1.0.1
```

Comment:
Can we avoid this extra package?

Reply:
Probably, but would need to create a basic yaml parser which is more complicated than it initially appears. frontmatter is a yaml wrapper that streamlines parsing metadata from a text file, like a markdown file in this case. It's being used to extract the metadata. Could probably look for the separators `---`, but these are also used as horizontal separators. Could probably look for the first line since HF complains when it's missing. Then look for the first line that appears afterwards and use that as a block. Extract the metadata from there and parse it accordingly. The HF API actually supports this already.